spec/support/trackjs_helpers.rb

```diff
@@ -0,0 +1,57 @@
+require 'securerandom'
+require 'faker'
+
+module TrackJSHelpers
+  def trackjs_error(params = {
+    message: nil,
+    timestamp: Time.now.strftime("%FT%T%:z"),
+    url: "https://www.remind.com",
+    application: "r101-frontend",
+    id: SecureRandom.uuid
+  })
+    trackJsUrl = "https://my.trackjs.com/details/#{params[:id]}"
+    params[:message] ||= Faker::Lorem.sentence
+
+    return {
+      "message" => params[:message],
+      "timestamp" => params[:timestamp],
+      "url" => params[:url],
+      "id" => params[:id],
+      "browserName" => "Chrome",
+      "browserVersion" => "49.0.2623",
+      "entry" => "window",
+      "application" => params[:application],
```

Comment:
I didn't see this in the docs: http://docs.trackjs.com/data-api/errors. Is it actually being returned or are the docs incorrect?